services/asset/nodes/parsers.go

```diff
@@ -76,19 +76,110 @@ func (node *Node) FromMap(nodeMap map[string]any) error {
 			}
 		}
 	}
-	node.NoradId = nodeMap["norad_id"].(string)
-	node.InternationalDesignator = nodeMap["international_designator"].(string)
+	var propertylist []Property[any]
+	if nodeMap["norad_id"] != nil {
+		noradInfo := Property[any]{}
+		noradInfo.Attributes.Type = "TEXT"
+		noradInfo.Attributes.Value = nodeMap["norad_id"].(string)
+		noradInfo.Name = "NORAD ID"
```

Comment:
It would be nice to have these defines once (see line 148 for the deuplicate) so we don't make mistakes